__tests__/fixtures/fakeContentCreator.js

```diff
@@ -1,11 +1,10 @@
 const { encrypt } = require('../../helpers/password');
 const mongoose = require('mongoose');
 
-module.exports = function makeFakeCreator() {
-  return {
-    email: "test1@mail.dk",
-    password: encrypt('ABC123456!'),
-    name: "test1 test1",
-    approved: true
-  }
-}
\ No newline at end of file
+module.exports = function makeFakeContentCreator(email = 'fake@gmail.com') {
+    return {
+        email: email,
+        password: encrypt('ABC123456!'),
+        name: 'Fake name',
```

Comment:
Does this model still only use name, or has this not been changed to firstname and lastname yet?

Reply:
it only uses name for now. we will update this file to contain first name and last name in a future sprint, but for now we only use name